### PR TITLE
Refactored search config to work efficiently on website

### PR DESCRIPTION
**Pull request by thisisobate**
Signed-off-by: thisisobate <obasiuche62@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [x] Change is not relevant to the end user.

## Changes
Refactored search config to work efficiently on website and made search work on home page

## Verification
Works on local

### CODE REVIEW

**Comment by thisisobate** [comment]
PTAL @bwplotka @squat

File: website/layouts/partials/searchConfig/docs.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"
+    src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"> docsearch({
+        apiKey: '649aaeb84811844d97cb5aa0398b94d9',
+        indexName: 'thanos',
+        inputSelector: '#searchInput',
+        algoliaOptions: { 'facetFilters': ["version:{{ $version }}"] },
+        debug: false // Set debug to true if you want to inspect the dropdown
+    });
+</script>
```
Comment (kakkoyun) [inline website/layouts/partials/searchConfig/docs.html:11]: nit

```suggestion
</script>

```

File: website/layouts/partials/searchConfig/docs.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{ $version := index (split .File.Path "/") 0 }}
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"
+    src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"> docsearch({
+        apiKey: '649aaeb84811844d97cb5aa0398b94d9',
+        indexName: 'thanos',
+        inputSelector: '#searchInput',
+        algoliaOptions: { 'facetFilters': ["version:{{ $version }}"] },
+        debug: false // Set debug to true if you want to inspect the dropdown
```
Comment (kakkoyun) [inline website/layouts/partials/searchConfig/docs.html:9]: nit 

```suggestion
        debug: false // Set debug to true if you want to inspect the dropdown.
```

File: website/layouts/partials/searchConfig/index.html [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{{ $tip := "tip"}}
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"
+    src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"> docsearch({
+        apiKey: '649aaeb84811844d97cb5aa0398b94d9',
+        indexName: 'thanos',
+        inputSelector: '#searchInput',
+        algoliaOptions: { 'facetFilters': ["version:{{$tip}}"] },
+        debug: false // Set debug to true if you want to inspect the dropdown
```
Comment (kakkoyun) [inline website/layouts/partials/searchConfig/index.html:9]: nit

```suggestion
        debug: false // Set debug to true if you want to inspect the dropdown.
```

File: website/layouts/partials/searchConfig/index.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{{ $tip := "tip"}}
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"
+    src="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js"></script>
+<script nonce="93HDK8392dfjh4Gb" type="text/javascript"> docsearch({
+        apiKey: '649aaeb84811844d97cb5aa0398b94d9',
+        indexName: 'thanos',
+        inputSelector: '#searchInput',
+        algoliaOptions: { 'facetFilters': ["version:{{$tip}}"] },
```
Comment (kakkoyun) [inline website/layouts/partials/searchConfig/index.html:8]: nit 
```suggestion
        algoliaOptions: { 'facetFilters': ["version:{{ $tip }}"] },
```

File: website/layouts/partials/searchConfig/index.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+        debug: false // Set debug to true if you want to inspect the dropdown
+    });
+</script>
```
Comment (kakkoyun) [inline website/layouts/partials/searchConfig/index.html:11]: nit

```suggestion
</script>

```

**Review by kakkoyun — APPROVED** [review]
Overall LGTM. Only some minor details.

**Comment by thisisobate** [comment]
Fixed! Thanks @kakkoyun

**Review by prmsrswt — APPROVED** [review]
LGTM!

**Pull request merged**